symmetric-io/src/main/java/org/jumpmind/symmetric/io/data/writer/DefaultDatabaseWriter.java

```diff
@@ -1075,6 +1075,7 @@ protected void bindVariables(Map<String, Object> variables) {
     protected List<String> getSqlStatements(String script) {
         List<String> sqlStatements = new ArrayList<String>();
         SqlScriptReader scriptReader = new SqlScriptReader(new StringReader(script));
+        scriptReader.setStripOutBlockComments(false);
```

Comment:
In order to support backwards compatibility I recommend adding a new parameter with default value of "true" in the symmetric-default.properties file (as well as in ParameterConstants.java)

Sample moniker: ddl.sync.routine.comments
Sample constant: public final static String DDL_SYNC_ROUTINE_COMMENTS="ddl.sync.routine.comments"

Reply:
The DefatulDatabaseWriter can access DatabaseWriterSettings (variable is writerSettings), so add a property like stripOutComments to it.  You can set it to the parameter value in AbstractDataLoaderFactory class buildParameterDatabaseWriterSettings() method.  Those parameters are prefixed with "dataloader", so maybe call it "dataloader.sql.event.strip.comments".

Reply:
Added new setting dataloader.sql.event.strip.comments with a default of true